minigalaxy/launcher.py

```diff
@@ -114,8 +129,24 @@ def get_scummvm_exe_cmd(game, files):
 
 
 def get_start_script_exe_cmd(game, files):
-    return [os.path.join(game.install_dir, "start.sh")]
+    command = game.get_info("command").split(" ")
+    var = game.get_info("variable").split(" ")
+    i = 0
+
+    exec_start = [os.path.join(game.install_dir, "start.sh")]
+
+    # Check if variable environment are set.
+    # Command can be empty
+    if var[0] != "":
+        while i < len(var):
+            if i < len(var):
+                exec_start.insert(i, var[i])
+                i = i + 1
+            else:
+                break
 
+    exec_start.extend(command)
+    return exec_start
```

Comment:
As above this can be done easier:
command_list = game.get_info("command").split()
var_list = game.get_info("variable").split()
exec_start = var_list + [os.path.join(game.install_dir, "start.sh")] + command_list
return exec_start

Reply:
Same here :D